table.cc

```diff
@@ -1666,17 +1666,20 @@ future<> table::generate_and_propagate_view_updates(const schema_ptr& base,
         tracing::trace_state_ptr tr_state,
         gc_clock::time_point now) const {
     auto base_token = m.token();
-    return db::view::generate_view_updates(
+    auto updates = co_await db::view::generate_view_updates(
             base,
             std::move(views),
             flat_mutation_reader_from_mutations(std::move(permit), {std::move(m)}),
             std::move(existings),
-            now).then([this, base_token = std::move(base_token), tr_state = std::move(tr_state)] (utils::chunked_vector<frozen_mutation_and_schema>&& updates) mutable {
-            tracing::trace(tr_state, "Generated {} view update mutations", updates.size());
-        auto units = seastar::consume_units(*_config.view_update_concurrency_semaphore, memory_usage_of(updates));
-        return db::view::mutate_MV(std::move(base_token), std::move(updates), _view_stats, *_config.cf_stats, std::move(tr_state),
-                std::move(units), service::allow_hints::yes, db::view::wait_for_all_updates::no).handle_exception([] (auto ignored) { });
-    });
+            now);
+    tracing::trace(tr_state, "Generated {} view update mutations", updates.size());
+    auto units = seastar::consume_units(*_config.view_update_concurrency_semaphore, memory_usage_of(updates));
+    try {
+        co_await db::view::mutate_MV(std::move(base_token), std::move(updates), _view_stats, *_config.cf_stats, std::move(tr_state),
+            std::move(units), service::allow_hints::yes, db::view::wait_for_all_updates::no);
+    } catch (...) {
+        // ignore
```

Comment:
I see you preserved ignoring exceptions without explaining why. Please follow up with an explanation about why it is correct to ignore errors here.